requirements.txt

```diff
@@ -25,7 +25,7 @@ docopt==0.6.2             # via hdfs
 elasticsearch==7.10.0     # via -r requirements.in
 enum34==1.1.10            # via google-cloud-tasks, grpcio, pyarrow
 fastavro==0.23.6          # via apache-beam
-git+git://github.com/oppia/firebase-admin-python@py2-emulator-support#egg=firebase-admin  # via -r requirements.in
+git+git://github.com/oppia/firebase-admin-python@03b0741100f9b521d093c14563310aea8904f851#egg=firebase-admin  # via -r requirements.in
```

Comment:
I thought you were planning to make this into a branch/release name, is it just that those changes haven't been pushed yet?

Reply:
No, this is how we will declare git dependencies. You must use the entire SHA-1 hash of the commit (40-character hexadecimal string).